src/qt/modaloverlay.h

```diff
@@ -45,6 +46,7 @@ public Q_SLOTS:
     QVector<QPair<qint64, double> > blockProcessTime;
     bool layerIsVisible;
     bool userClosed;
+    QPropertyAnimation animation{};
```

Comment:
nit, m_animation, m_slide_animation or something like that. Also, could drop `{}`?

Reply:
Sure! Going to fix.

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/17968#issuecomment-621398261).